### DISCUSSION TITLE
Using OTD without systemd

### DISCUSSION THREAD

**Discussion by Belamat** [OpenTabletDriver/OpenTabletDriver#2512] [Q&A] [answered]
Hi, im on linux mx on one machine, a distro that doesnt comes with systemd by default, could some one help me or guide me on how to set OTD without systemd, is it possible?

Thanks!

**Comment by jamesbt365** [comment, marked as answer]
`dotnet ./OpenTabletDriver.Daemon.dll` in the directory you have otd and `dotnet ./OpenTabletDriver.UX.Gtk` if you want the ui
